azure-pipelines.yml

```diff
@@ -102,19 +148,17 @@ jobs:
       azureContainerRegistry: '{"loginServer":"gpitfuturestestacr.azurecr.io", "id" : "/subscriptions/7b12a8a2-f06f-456f-b6f9-aa2d92e0b2ec/resourceGroups/gpitfutures-test-rg-acr/providers/Microsoft.ContainerRegistry/registries/gpitfuturestestacr"}'
       dockerComposeFile: '**/docker-compose.yml'
       action: 'Build services'
-      additionalImageTags: '$(Build.BuildNumber)'
+      additionalImageTags: '$(semVer)'
```

Comment:
Is it possible to make it so that latest is only updated when the branch is development / master? Not sure of the behaviour of this task.

Reply:
👍